resources/startPage.html.template

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; img-src {{ cspSource }} file:; script-src 'nonce-{{ nonce }}'; font-src {{ codiconsFontUri }}; style-src {{ cspSource }} {{ codiconsStyleUri }} 'nonce-{{ nonce }}';" />
+    <link href="{{ codiconsStyleUri }}" rel="stylesheet" />
+    <title>Docker - Get Started</title>
+
+    <style nonce="{{ nonce }}">
+        #titleBar {
+            align-items: center;
+        }
+
+        .icon .codicon {
+            font-size: 70px;
+            padding: 5px;
+        }
+
+        #dockerIcon {
+            width: 70px;
+            padding: 5px;
+        }
+
+        #dockerIcon path {
+            fill: var(--vscode-editor-foreground);
+        }
+
+        a.icon:hover {
+            color: var(--vscode-editor-foreground);
+        }
+
+        div .container {
+            display: flex;
+        }
+
+        div .row {
+            display: flex;
+            flex-direction: row;
+            justify-content: flex-start;
+            align-items: flex-start;
+        }
+
+        div .cell {
+            margin: 10px;
+        }
+
+        .linkheading {
+            text-decoration: none;
+            color: var(--vscode-editor-foreground);
+        }
+    </style>
+</head>
+
+<body>
+    <div class="container">
+        <div class="row" id="titleBar">
+            <div class="cell">
+                <a href="https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker"><img src="{{ dockerIconUri }}" /></a>

```

Comment:
Should all of the icon-based links have `alt` or `a11y` tags for screen readers?

Reply:
Yes, for sure!